api/cases/case/templates/divorce/financialremedy.js

```diff
@@ -44,10 +44,6 @@ module.exports = {
                                 {
                                     label: 'Case number',
                                     value: '$.id'
```

Comment:
this need to also be comment out and have the correct Divorce case number associated with it